renter/download.go

```diff
@@ -0,0 +1,162 @@
+package renter
+
+import (
+	"compress/zlib"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/user"
+	"path"
+	"skybin/core"
+	"skybin/provider"
+)
+
+func (r *Renter) Download(fileId string, destPath string) error {
+	f, err := r.Lookup(fileId)
+	if err != nil {
+		return err
+	}
+	if f.IsDir {
+		return errors.New("Folder downloads not supported yet")
+	}
+
+	// Download to home directory if no destination given
```

Comment:
Other applications I've seen (Slack, Firefox, Chrome) place downloads in the `Downloads` folder in the user's home directory. In fact, I think Slack just always places them there. That would probably be amore reasonable default for us.

Reply:
Agree